test/types/serverFactory.test-d.ts

```diff
@@ -28,11 +28,11 @@ const serverFactory: FastifyServerFactory<http.Server> = (handler, opts) => {
 const customServer = fastify<http.Server, CustomIncomingMessage, CustomServerResponse>({ serverFactory })
 
 customServer.get('/', function (request, reply) {
-  if (request.fakeMethod) {
-    expectType<CustomType>(request.fakeMethod())
+  if (request.raw.fakeMethod) {
```

Comment:
why was this moved to `request.raw`?

Reply:
because `FastifyRequest` doesn't inherit nodejs request any more.